en/components/tree-grid.md

```diff
@@ -24,6 +24,20 @@ Similarly to the Grid, the Tree Grid is essentially a repeater of columns and ro
 
 After detaching, you may add the number of headers you need to show all the dimensions of your data and as many records as you want to show in your design simply by duplicating one of the existing rows of data.
 
+## Using the Tree Grid in Adobe XD
+
+In Adobe XD, after you choose one of the three [predefined grids](grid-display-density.md) and drag it to your artboard, you can see in the layers panel that we use `Repeat Grid` for both the Header and the Body to speed configuration up. Unlike in Sketch in Adobe XD destroying the symbol is not necessary. You should see the following in your layers panel:
+
+| Layer                        | Use                                                                                                                                                  |
+| ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 🚫 metadata | A special locked layer starting with a prohibited icon. This layer is required by the code generation and you should avoid deleting or modifying it. |
+| Header                       | Contains all header cells: the one relating to the TreeColumn body cells and the rest grouped in a Repeat Grid                                                                                                                 |
+| Body                         | Contains all body cells: the TreeColumn cells grouped in a Repeat Grid and the rest grouped in another Repeat Grid                                                                                                                  |
+
+When you select a Repeat Grid layer, you may alter the number of the repeated cells you need. Just drag the green handles at the end of the right or bottom side to show all the dimensions of your data and insert as many visible records as you want to show in your design. To change a particular column header and body cells types of data or column width, you have to select the Header Repeat Grid layer and click `Ungroup Grid` in the right panel and do the same with the Body Repeat Grid. This will ungroup the respective replicated grid cells, and you will be able to work on them separately to achieve the desired design.
```

Comment:
```suggestion
When you select a Repeat Grid layer, you may alter the number of the repeated cells you need. Just drag the green handles at the right or bottom edge to show all the dimensions of your data and insert as many visible records as you want to show in your design. To change a particular cell type or column width, you have to select the Repeat Grid layer and click `Ungroup Grid` in the right panel. This will allow you to work the cells separately to achieve the desired design.
```